src/main/java/iudx/catalogue/database/MongoDB.java

```diff
@@ -314,6 +401,25 @@ public void updateSchema(Message<Object> message) {
 
   }
 
+  private void decItemsTags(JsonArray tags) {
```

Comment:
Update the method names, call back handler appropriately. Also verify the handler for write failure before response.